test/Driver/response-file-crash.swift

```diff
@@ -0,0 +1,8 @@
+// Windows returns a negative exit code from -debug-crash-immediately so not
+// considers that to be a crash (requiring --crash) instead of an abnormal exit
```

Comment:
Hm. The driver is supposed to be catching crashes when not doing an `exec`. Is it possible the decision of whether to exec or not is different on Windows vs. other platforms?

Reply:
What do you mean by catching crashes when not doing an `exec`?

Reply:
The TaskQueue implementation has a callback for when a process crashes, and is supposed to exit with a normal error when that happens.

Reply:
From what I've seen, on Unix, if a process exits with an uncaught signal, the driver catches that and exits with a positive exit code (254 it seems).

On Windows, signals aren't really a thing, so the signal handler only gets called if llvm:sys::Wait fails due to say, a malformed argument. Otherwise, it simply bubbles up the return code to the driver and the driver exits with it. I think the closest Windows thing would be [checking the high bits](https://docs.microsoft.com/en-us/windows/desktop/wmisdk/wmi-return-codes) of the 32bit exit code. 10 is a warning, 11 is an error which llvm respects. However, that's not really a signal and I don't think it makes too much sense to have that call the signalled callback.

Reply:
I think the "signalled" callback really is supposed to be "crashed". It controls things like whether temporary files are cleaned up or left in place for debugging purposes.